android/anc/src/androidTest/java/org/smartregister/fhircore/anc/ui/family/details/FamilyDetailScreenTest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.anc.ui.family.details
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.Test
+import org.smartregister.fhircore.engine.ui.theme.AppTheme
+
+class FamilyDetailScreenTest {
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Ignore("will be worked later")
```

Comment:
I think we can remove this test class, integration testing is not working yet

Reply:
done.